--- webapp-pod.yaml	2021-07-01 14:25:30.183276800 +0200
+++ webapp-pod.yaml.patched	2021-07-08 14:10:07.489967800 +0200
@@ -50,7 +50,11 @@
           #command: [ "/bin/sh", "-c", "--" ]
           #args: [ "while true; do sleep 30; done;" ]
           command: [ "/bin/sh", "-c", "--" ]
-          args:  [ "sleep 60; node setup.js; node server.js;" ]
+          args:  
+            - sed -E "/createPasswordHashAndEncKeys/s/'admin'/'${FRONTEND_ADMIN_PW}'/g" -i /webapp/backend/adapter/userManagement/v1/UserManagementAdapter.js;
+              sleep 60;
+              node setup.js;
+              node server.js;
           envFrom:
             - configMapRef:
                 name: proxy-environment-variables
